classes/class05/HwApp/public/javascripts/main.js

```diff
@@ -0,0 +1,88 @@
+var $tvveetForm = $('.tvveet-form');
+var $tvveetsGroup = $('.tvveet');
+var $usersGroup = $('.user-btn');
+
+var user_id = $tvveetForm.attr('user_id');
+console.log($usersGroup);
+$.material.init();
+
+function enable_deletes (){
+    $tvveetsGroup.each(function (){
+        var $tvveet = $(this);
+        console.log($tvveet.attr('author_id'));
+        if($tvveet.attr('author_id') == user_id){
```

Comment:
Would have been better to not do this on the client side (I can still send a post request and delete someones tvveet) and to also not even have the button on those tvveets which you don't want me to delete.